tests/Dfc.CourseDirectory.Core.Tests/DataManagementTests/FileUploadProcessorTests.cs

```diff
@@ -145,5 +146,180 @@ public async Task FileMatchesSchema_RowHasIncorrectColumnCount_ReturnsInvalidRow
             // Assert
             result.Should().Be(FileMatchesSchemaResult.InvalidRows);
         }
+
```

Comment:
Each of these tests need to create a file in the valid schema to be valid. Using `DataManagementFileHelper.CreateCourseUploadRows(learnAimRef, rowCount)` may be easier.